terminus-ui/popover/src/popover.component.md

```diff
@@ -0,0 +1,95 @@
+<h1>Popover</h1>
+
+Popover component is designed to pop up content based on user trigger.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic info](#basic-info)
+- [Basic Usage](#basic-usage)
+  - [Position](#position)
+  - [Hide on Blur](#hide-on-blur)
+  - [Open on load](#open-on-load)
+- [Events](#events)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Basic info
+
+We wrap our popover trigger/component with a third party library, named [popper.js](https://github.com/popperjs/popper.js).
+`popper.js` is not a required import for ui library but if it's not imported and consumers try to use popover component, an error will show up. 
+
+## Basic Usage
+
+Define a popover trigger and popover content:
+
+```html
+<button tsPopoverTrigger="popper1">Click me!</button>
+
+<ts-popover #popper1>
+  <h1>My Title</h1>
+  <p>Other random content.</p>
+</ts-popover>
+```
+
+### Position
+
+Defines where the popover shows relative to current element.
+
+```html
+<button 
+  tsPopoverTrigger="popper1"
+  [position]="top"
+>Click me!</button>
+
+<ts-popover #popper1>
+  <h1>My Title</h1>
+  <p>Other random content.</p>
+</ts-popover>
+```
+
+It defaults to `bottom`. For all available positions, please reference [here](https://github.com/FezVrasta/popper.js/blob/master/packages/popper/src/methods/placements.js).
```

Comment:
Let's keep all our links defined at the bottom of the file to follow existing patterns. Probably should change the link text to something like `please reference the [popper.js documentation]` etc